README.md

```diff
@@ -33,7 +33,7 @@ In all cases, you will want to perform the following steps:
 
 This translates into the following steps:
 
-1. Inspect the **shared/params.json** file and change it if needed
+1. Inspect the **shared/params.json** file and change it if needed (if you have more than 16GB RAM, feel free to increase `memory` and `count` values)
```

Comment:
I'd go the other way about it with a default appropriate for a better machine and a file with alternate minimum values they can swap if their specs are less.

Reply:
Done.